### PR TITLE
MNIST Inference Tutorial using netx

### PR DESCRIPTION
**Pull request by R-Gaurav**
Issue Number: #849

Objective of pull request: Contributing an inference tutorial on classifying MNIST using `netx` (by loading a trained `slayer`-network)

## Pull request checklist

Your PR fulfills the following requirements:
- [x] [Issue](https://github.com/lava-nc/lava-dl/issues) created that explains the change and why it's needed
- [ ] Tests are part of the PR (for bug fixes / features)
- [ ] [Docs](https://github.com/lava-nc/docs) reviewed and added / updated if needed (for bug fixes / features)
- [x] PR conforms to [Coding Conventions](https://lava-nc.org/developer_guide.html#coding-conventions)
- [ ] [PR applys BSD 3-clause or LGPL2.1+ Licenses](https://lava-nc.org/developer_guide.html#add-a-license) to all code files
- [ ] Lint (`flakeheaven lint src/lava tests/`) and (`bandit -r src/lava/.`) pass locally
- [ ] Build tests (`pytest`) passes locally

## Pull request type

 

 

Please check your PR type:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [x] Other (please describe): 
Contributing a tutorial in lava-dl/tutorials/lava/lib/dl/netx/mnist

## What is the current behavior?

- N/A

## What is the new behavior?

- N/A

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Supplemental information
An extended tutorial on training and inference both exists [here](https://r-gaurav.github.io/2024/04/13/Lava-Tutorial-MNIST-Training-on-GPU-and-Evaluation-on-Loihi2.html).

### CODE REVIEW

File: tutorials/lava/lib/dl/netx/mnist/utils.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import numpy as np
```
Comment (bamsumit) [inline tutorials/lava/lib/dl/netx/mnist/utils.py:1]: Add license header
```python
# Copyright (C) 2024
# SPDX-License-Identifier: BSD-3-Clause
```

**Review by bamsumit — COMMENTED** [review]
@R-Gaurav looks great. Only one comment on adding license header.

File: tutorials/lava/lib/dl/netx/mnist/utils.py (hunk 1 above)
Comment (R-Gaurav) [reply tutorials/lava/lib/dl/netx/mnist/utils.py:4]: Done!

**Comment by R-Gaurav** [comment]
Thank you for reviewing @bamsumit ! I have resolved your comment, as well as freshly evaluated the `run.ipynb` on CPU and Loihi-2.

**Review by bamsumit — APPROVED** [review]

**Comment by R-Gaurav** [comment]
Thank you @bamsumit for approving and merging :)

**Pull request merged**